### ISSUE TITLE
Incorrect peer dep

### ISSUE THREAD

**Issue by richardsimko** [davesag/sequelize-test-helpers#356]
If a project where this package is used also has the latest `sinon` installed (11.1.1) which should be OK according to the dependencies the following error appears on `npm@7`:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: woc-api@0.0.1
npm ERR! Found: sinon@11.1.1
npm ERR! node_modules/sinon
npm ERR!   dev sinon@"11.1.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer sinon@"^10.0.0" from sequelize-test-helpers@1.4.1
npm ERR! node_modules/sequelize-test-helpers
npm ERR!   dev sequelize-test-helpers@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
```

Would it be OK to upgrade the peer dependency as well in this project since the dependency has been upgraded?

**Comment by davesag** [comment]
Thank I will update and get back to you shortly.  Sorry for the issue.

**Comment by davesag** [comment]
Thanks @richardsimko I have fixed this and released an update - Version 1.4.2

Sorry for the inconvenience.
